doc/source/ray-contribute/writing-code-snippets.rst

```diff
@@ -0,0 +1,189 @@
+.. _writing-code-snippets:
+
+==========================
+How to write code snippets
+==========================
+
+Docstrings and user guides should contain examples that illustrate Ray's APIs.
```

Comment:
It is hard to emphasize the importance of this -- I would state it here again and examples are very important for users to understand how something is used, and being able to copy and paste it and adapt it to their own needs